robot/rospackages/src/mcu_control/scripts/MockPdsNode.py

```diff
@@ -0,0 +1,287 @@
+#! /usr/bin/env python3
+
+import time
+import rospy
+import sys
+import random
+from std_msgs.msg import Float32
+from geometry_msgs.msg import Point
+from mcu_control.msg import ThermistorTemps, Voltage, Currents
+from decimal import *
```

Comment:
Thank you for the comments! 
1. I'll look into PEP8 to reformat!

2. It is possible to change the `rospy.Rate` to something more appropriate if the values are fluctuating too quickly. 

3. How long have you let the temperature to rise/fall? It seems to be fine on my setup; however, the rates of rising and decreasing are initially quite slow (`PDS_mock_voltage_rate` set to 0.04, `PDS_mock_temp_rate` set to 1 and `PDS_mock_current_rate` set to 0.025). Because of this, it takes a while for the values to reach their target. Next, the time it takes to rise to a particular value can be decreased by changing the increment rates (`PDS_mock_voltage_rate`, `PDS_mock_temp_rate`, `PDS_mock_current_rate`) as opposed to defining a specific time to reach the target value. This is done by setting a float value, and originally the values are set to 0.04, 1 and 0.025 respectively. However, these values can be made larger so the time to reach the target value is decreased. 

4. I will take a look into the negative value, I can add a condition to avoid this.

5. Would you like me to create a dedicated wiki page for the simulation?